src/webviews/rebase/rebaseEditor.ts

```diff
@@ -657,7 +658,7 @@ async function parseRebaseTodo(
 			author: commit.author.name,
 			committer: commit.committer.name,
 			date: commit.formatDate(defaultDateFormat),
-			dateFromNow: commit.formatDateFromNow(),
+			dateFromNow: defaultDateStyle === 'absolute' ? commit.formatDate() : commit.formatDateFromNow(),
```

Comment:
Looks like `commit.formattedDate` should handle the dateStyle formatting without needing to check the style manually.

See: https://github.com/gitkraken/vscode-gitlens/blob/dd8c3497ff51367b37a0be674643d2d8c96f32ed/src/git/models/commit.ts#L136

Reply:
So right; I completely missed that, thanks for the heads up!

I made the requested change, and it all checks out!